views/library/track_overview_panel.html

```
@@ -16,10 +16,25 @@
-      <img src="/ui/tag/edit/{{.Track.ID}}/artwork" alt="Album art" class="w-full object-cover max-h-64"
+      <img src="/tag/{{.Track.ID}}/artwork" alt="Album art" class="w-full object-cover max-h-64"
            onerror="this.parentElement.style.display='none'">
     </div>
 
+    <!-- Audio Player -->
+    <div class="audio-player px-4 py-3 border-b border-gray-200 dark:border-gray-800 flex items-center gap-2 shrink-0">
+      <button type="button" onclick="toggleAudioPlayer(this)"
+              class="play-btn flex-shrink-0 w-7 h-7 rounded-full bg-gray-200 dark:bg-gray-700 hover:bg-gray-300 dark:hover:bg-gray-600 flex items-center justify-center transition-colors">
+        <i class="fas fa-play fa-xs text-gray-700 dark:text-gray-200"></i>
+      </button>
+      <input type="range" min="0" max="100" value="0" step="0.1"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)